Projections/WinUI/WinUI.csproj

```diff
@@ -22,28 +15,8 @@
     </PackageReference>
   </ItemGroup>
 
-  <Target Name="GenerateProjection" Condition="'$(GenerateTestProjection)' == 'true'">
-    <PropertyGroup>
-      <!--PkgMicrosoft_WinUI is set in in obj\*.csproj.nuget.g.props with TargetFramework condition, doesn't support multi-targeting-->
-      <PkgMicrosoft_WinUI Condition="'$(PkgMicrosoft_WinUI)'==''">$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'microsoft.winui', '$(MicrosoftWinUIVersion)'))</PkgMicrosoft_WinUI>
-    </PropertyGroup>
-    <ItemGroup>
-      <!--Explicitly reference WinUI winmds from TFM uap10.0-->
-      <ReferenceWinMDs Include="$(PkgMicrosoft_WinUI)/**/*.winmd" />
-      <ReferenceWinMDs Include="@(ReferencePath)" Condition="'%(ReferencePath.WinMDFile)' == 'true'" />
-    </ItemGroup>
-    <PropertyGroup>
-      <CsWinRTVerbosity>high</CsWinRTVerbosity>
-      <CsWinRTResponseFile>$(GeneratedFilesDir)cswinrt_winui.rsp</CsWinRTResponseFile>
-      <CsWinRTCommand>$(CsWinrtExe) %40"$(CsWinRTResponseFile)"</CsWinRTCommand>
-    </PropertyGroup>
-    <PropertyGroup>
-      <CsWinRTParams>
--verbose
--target $(TargetFramework)
--in 10.0.18362.0
--in @(ReferenceWinMDs->'"%(FullPath)"', ' ')
--out "$(GeneratedFilesDir.TrimEnd('\'))"
+  <PropertyGroup>
+    <CsWinRTFilters>
```

Comment:
We're not using the CsWinRTIncludes tag here, is it interchangeable with this format using CsWinRTFilters ?

Reply:
right, this demonstrates the more complex use of CsWinRTFilters, versus the simpler CsWinRTIncludes (and, optionally, CsWinRTExcludes)